src/skmultiflow/trees/hoeffding_adaptive_tree_regressor.py

```diff
@@ -86,6 +87,7 @@ class HoeffdingAdaptiveTreeRegressor(HoeffdingTreeRegressor):
     # ======================================================
     # == Hoeffding Adaptive Tree Regressor implementation ==
     # ======================================================
+    ERROR_WIDTH_THRESHOLD = 300
```

Comment:
Please make this constant private.

Reply:
Addressed in commit [f5838ee](https://github.com/scikit-multiflow/scikit-multiflow/pull/220/commits/f5838ee9b96b27c674a13a529e065d3b91c38aa2).